docs/source/references/functional.rst

```diff
@@ -281,17 +281,6 @@ ssim [func]
 .. autofunction:: torchmetrics.functional.ssim
     :noindex:
 
-
-***
-NLP
```

Comment:
shall we call it NLP or Text?

Reply:
I am not sure about this. NLP also includes speech processing. So, if we are to add those metrics as well then we can call it NLP.

Reply:
well, speech processing you mean conversion from audio > text and back, right?
bu then you still shall measure the quality against each independently as your prediction and target are always other audio or text, right? so we can split NLP as text and audio... :rabbit: 
cc: @SkafteNicki @maximsch2

Reply:
I prefer text, as the data modality that bleu works on is text, similar to how we have grouped other metrics based on data modality.
Also, this does not matter for the end users, as all modular metrics can just be imported with `from torchmetrics import *` and functional `from torchmetrics.functional import *